### ISSUE TITLE
padj 0.5 is not FDR 0.05?

### ISSUE THREAD

**Issue by jcglaubitz** [ACSoupir/Bioinformatics_RNASeq#2]
Hi Alex,
Your YouTube Bioinformatics series is awesome, thank you!!!

I noticed one potential issue, however.  Shouldn't this be `padj<0.05`?
`res1 = mutate(res1, sig=ifelse(res1$padj<0.5, "FDR<0.05", "Not Sig"))`

Thanks,
Jeff

**Comment by ACSoupir** [comment]
Good catch! As long as the 2 are the same then it wouldn't be an issue. Would just have to justify the choice of a high FDR but you are right, it should be `padj<0.05`.
